test/fixtures/git/fixture_git.js

```diff
@@ -0,0 +1,69 @@
+var nock = require('nock');
+
+function _cacheKeyUrl(n) {
+  return '/box/srv/1.1/dat/log/read';
+}
+
+var cacheReplies = {
+  headers : { 'Content-Type': 'application/json' },
+  deliver : function() {
+    return { "cacheKey": "somechachekey" };
+  },
+  cacheKey : function() {
+    return [{
+      "action": {},
+      "cacheKey": "somechachekey",
+      "error": "",
+      "log":  [ 'Cordova App Git pull finished sucessfully .' ],
+      "status": "pending"
+    }];
+  },
+  success : function() {
+    return [{"action":{},"cacheKey":"somechachekey","error":"","log":['Cordova App Git pull finished sucessfully .'],"status":"complete"}];
+  }
+};
+
+module.exports = nock('https://apps.feedhenry.com')
```

Comment:
is this logic uses else where in the code base? Would prefer to see it as part of the test definition file rather than a fixtures file. If it is used across many tests then fair enough